bucket/confuser-ex.json

```diff
@@ -0,0 +1,16 @@
+{
+    "version": "1.5.0",
+    "description": "An open-source protector for .NET applications",
+    "homepage": "https://github.com/mkaring/ConfuserEx",
+    "license": "MIT",
+    "url": "https://github.com/mkaring/ConfuserEx/releases/download/v1.5.0/ConfuserEx.zip",
+    "hash": "c1d1b6d359fed0da7c2d42d63452cca7a9a1849d32aca780050cd0ee80a778d3",
+    "bin": [
+        "ConfuserEx.exe",
```

Comment:
Does not make sense to flood path with GUI application, which does not support parameters.